scripts/mdbook-forc-documenter/src/plugins.rs

```diff
@@ -0,0 +1,28 @@
+use std::fs;
+use std::path::PathBuf;
+
+fn get_sway_path() -> PathBuf {
+    let mut curr_path = std::env::current_dir().unwrap();
+    loop {
+        if curr_path.ends_with("sway") {
+            return curr_path;
+        }
+        curr_path = curr_path.parent().unwrap().to_path_buf()
+    }
+}
+
+pub fn plugin_commands() -> Vec<String> {
+    let plugins_dir = get_sway_path().join("forc-plugins");
+    let mut plugins: Vec<String> = Vec::new();
+
+    for entry in fs::read_dir(&plugins_dir)
+        .expect("Failed to read plugins directory")
+        .flatten()
+    {
+        let path = entry.path();
```

Comment:
We should at least check that the `path` is 1. a directory and 2. it starts with `forc-`. If it doesn't meet these criteria, we should `continue` onto the next entry (and not `unwrap` or `panic!`).

Otherwise we will run into issues if e.g. we decide to put a `README.md` in this directory to explain what crates are allowed in here, or a developer's editor puts some temporary file in here, etc.